codes/quantum/qudits/HaydenMay.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: hayden_may
+physical: qudits
+logical: qudits
+
+name: 'Hayden May, Summoring information in spacetime'
+introduced: '\cite{arxiv:1210.0913}'
+
+description: '|
+It is well-known that quantum information, which is usually presented as a state of a qubit

```

Comment:
This is a nice layperson intro, but this part of the zoo is for people familiar with basics of QEC. So start with a sentence saying these are codes used in the Hayden-May protocol which is used for ... (propagating information in spacetime?). Then continue with the nice description you formulated, but edit it so that you it is clear where, how, and why the error correcting codes feature.